images/devtools-terraform-v1beta1/context/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1.3
 FROM docker.io/safewaters/docker-lock:latest@sha256:432d90ddc2891f4845241adc63e5eef2dd1486fa14ea7882433cbd3f8ed64622 AS docker-lock
 FROM ghcr.io/terraform-linters/tflint-bundle:v0.34.1.2@sha256:25889a5d3738a97ba1fa1e72bdc1b05dd1e7087c58213597cb0e1ed30f52615a AS tflint
-FROM docker.io/aquasec/tfsec:v1.1@sha256:c867867e5eb038f8c3e9f0cf698c7b6158006ff68088273934718060c8c7002b AS tfsec
+FROM docker.io/aquasec/tfsec:v1.18.0@sha256:1260c09ff8d48d44be739418f8141800c4607c78c940d4acd9602dd50c7215c3 AS tfsec
```

Comment:
@aucampia maybe we can just use the latest version?

Reply:
I'm fire with this as is, can change if you really want but if dependabot updates it then I don't see a problem.